tensorflow_model_optimization/python/core/quantization/keras/prune_preserve/BUILD

```diff
@@ -0,0 +1,56 @@
+package(default_visibility = [
+    "//tensorflow_model_optimization:__subpackages__",
+])
+
+licenses(["notice"])  # Apache 2.0
+
+py_library(
+    name = "prune_preserve",
+    srcs = [
+        "__init__.py",
+    ],
+    srcs_version = "PY3",
+    deps = [
+        ":default_8bit_prune_preserve_quantize_scheme",
+    ],
+)
+
+py_library(
```

Comment:
Would make the registry and any libraries that don't need to be used externally private in terms of visibility.

Reply:
Done